content/_data/changelogs/weekly.yml

```diff
@@ -6,36 +6,40 @@
   date: 2017-02-19
   changes:
     - type: bug
-      message: Update Groovy to 2.4.8 to address memory leak issue
+      message: >
+        Update Groovy to 2.4.8 to address memory leak issue.
+        <strong>Do not use this version if you are running Pipeline builds.</strong>
       issue: 33358
+      references:
+        - issue: 42189
```

Comment:
Unless both issues are in `references`, they'll be listed separately (ignore CSS weirdness):

![screen shot](https://cloud.githubusercontent.com/assets/1831569/23240563/72272ca8-f96e-11e6-8814-fe532c9006c0.png)